utils/coreutils/tableutils.go

```diff
@@ -88,28 +90,58 @@ var DefaultMaxColWidth = 25
 // │ No customers were found │
 // └─────────────────────────┘
 func PrintTable(rows interface{}, title string, emptyTableMessage string, printExtended bool) error {
+	tableWriter, _, err := PrepareTable(rows, emptyTableMessage, printExtended)
+	if err != nil || tableWriter == nil {
+		return err
+	}
+
 	if title != "" {
 		fmt.Println(title)
 	}
 
-	tableWriter := table.NewWriter()
-	tableWriter.SetOutputMirror(os.Stdout)
-
 	if IsTerminal() {
 		tableWriter.SetStyle(table.StyleLight)
 	}
-
 	tableWriter.Style().Options.SeparateRows = true
+	tableWriter.SetOutputMirror(os.Stdout)
+	tableWriter.Render()
+	return nil
+}
+
+// Creates a table similar to PrintTable, but returns it as a slice of rows.
+// Each row is represented as a key:value where key is the column name and value is the corresponding value in the table.
+// For example, the following table:
+// ┌──────┬─────┐
+// │ Name │ Age │
+// ├──────┼─────┤
+// │ Gai  │ 350 │
+// ├──────┼─────┤
+// │ Noah │ 21  │
+// └──────┴─────┘
+// Becomes:
+// {[{"Name": "Gai", "Age": "350"}, {"Name": "Noah", "Age": "21"}]}
+func CreateJsonTable(rows interface{}) ([]map[string]interface{}, error) {
```

Comment:
The new simple-json format shouldn't be affected by future changes to the table format.
We should therefore probably remove this function (which creates the simple-json output) out of the tableutils.go, which includes the code for creating the tables.

Reply:
Will be fixed in a follow-up PR